apps/etterlatte-behandling/src/main/kotlin/behandling/etteroppgjoer/EtteroppgjoerRoutes.kt

```diff
@@ -104,6 +105,17 @@ fun Route.etteroppgjoerRoutes(
                     }
                 call.respond(response)
             }
+
+            post("har_mottatt_ny_informasjon") {
```

Comment:
```suggestion
            post("har-mottatt-ny-informasjon") {
```

vet vi ikke er konsekvent på det over alt, men bindestrek er imo nicere i url'er